ResourceStatusSystem/Command/PropagationCommand.py

```diff
@@ -39,7 +39,10 @@ def doCache( self ):
         if not status[ 'OK' ]:
           return status
 
-        statusList.append( status['Value'][0][0] )
+        if status['Value']:
```

Comment:
This changes the behaviour no ? Is it to avoid the error when we just define a new element that RSS has not yet fetched ?